docs/ref/contrib/gis/tutorial.txt

```diff
@@ -53,13 +53,19 @@ Create a Spatial Database
 
 First, create a spatial database for your project.
 
-If you are using PostGIS, create the database from the :ref:`spatial database
-template <spatialdb_template>`:
+If you are using PostGIS and an earlier version of PostgreSQL than 9.1, you will
```

Comment:
In which case, I'm not sure the special mention about how to create a database in PostgreSQL is needed (given the other databases aren't mentioned). What do you think?